tests/test_contrib.py

```diff
@@ -4,34 +4,91 @@
 except ImportError:
     pass
 
+import pytest
+
+from klaus.contrib import app_args
 from .utils import *
 from .test_make_app import can_reach_unauth, can_push_auth
 
 
+def clear_env():
+    for var in list(os.environ):
```

Comment:
Had to use list() because os.environ becomes a generator in py3, which should not be modified while iterating over it.

But yes, I'll use monkeypatch instead, thanks.